integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -158,6 +163,8 @@ def __init__(
         streaming_callback: Optional[StreamingCallbackT] = None,
         boto3_config: Optional[Dict[str, Any]] = None,
         tools: Optional[Union[List[Tool], Toolset]] = None,
+        *,
+        prompt_router_arn: Optional[Secret] = None,
```

Comment:
Just to double check, does `prompt_router_arn` need to be a Secret? What was the motivation to make it a secret?